### PR TITLE
Feature 1 branch

### PR DESCRIPTION
**Pull request by Osoro254Alex**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.md
```
Comment (ichala) [inline .gitignore:1]: - [ ] great job creating `.gitignore` file kindly add the files that needs to be ignored by GitHub which are :

1. node_modules
2. test.md

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Osoro254Alex , 

Good job so far! 👏👏👏
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- index.html and style.css are well implemented. ✔️
- Good Commit messages  ✔️

 ## Required Changes ♻️

- [ ] your project is missing linters which is a tool that makes your code clean and understandable by everyone, kindly set up your linters following these steps [here](https://github.com/microverseinc/linters-config).

- [ ] The README file is a guide that gives users a detailed description of a project you have worked on, so kindly add a readme file with descriptive details about your project you can use this [template](https://github.com/microverseinc/readme-template)

- [ ] the branch should have a name of a feature or action (e.g. 'set-up-project' or 'create-project-files' etc ...) kindly rename the current one which is `feature-1` .

- [ ]  Pull Request should have a descriptive title telling you what changes are inside. kindly Rename your PR title to be more meaningful

- [ ] Pull Request should have a summary describing introduced changes in more detail. currently, your PR description is empty, please try to fill it with the actions that you implemented , for example : 

> 1.  Create index.html
> 2.  Setup Linters
> 3. Create .gitignore file
> 4. etc ...


 
- [ ]  _**_Check The Comments Below_**_

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+</head>
+<body>
+    <header>
+        <h1>
+            Hello Mcroverse!
+        </h1>
+    </header>
+    
+</body>
+</html>
```
Comment (Kamba56) [inline index.html:18]: [Optional] Consider using indentation of two spaces for readability as this is the convention in Microverse community.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Osoro254Alex , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Professional readme file
- [ ] Meaningful commit messages

 ## Required Changes ♻️

- [ ] Kindly set up linters for your project. To set up linters in your project, copy the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) folder in the linters-config repo into your repository. Then open terminal and run `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` to set up stylelint and then run `npm install --save-dev hint@7.x` to set up webhint.
- [ ] Please always use a [feature or action](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches) as your branch name in future projects since renaming your branch in this repository will close this pull request.
- [ ] Kindly rename your PR title to be more meaningful. PR title should be a short summary of changes in the pull request. To edit PR title, click on the `edit` button next to the title.
- [ ] Please add a brief summary of changes implemented in this pull request in your PR description. To this please click on the hamburger icon and then click on edit. See picture below.
ch


_Check the comments under the review for more suggestions._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [Optional] It's better to follow commit message standards by making them capitalized and in the imperative form (Example: 'Modify readme file' instead of 'modified readme'). This way, your commit history will be more readable and more professional ✨

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Osoro254Alex, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- [ ] No linters errors ✔️ 

 ## Required Changes ♻️

- [ ] Please as mentioned by the previous reviewers, your Pull Request should have a descriptive title telling you what changes are inside for example instead of `Feature 1 branch` you can write `Project: Hello Microververse` or something else but descriptive 🙃 🙃
- [ ] Also as mentioned by the previous reviewers, the Pull Request should have a summary describing introduced changes in more detail, please kindly fix it 😁 😁.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@cynthiainga @ichala @Kamba56 
Added a correction on h1 Title

**Review by zieeco — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST  
 Hi @Osoro254Alex 
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
 * [ ]  note: **_kindly do not close or merge the pull request until you get approved by one of the reviewers, because we won't be able to use GitHub tools to provide you with the requested changes and feedback, kindly create a new PR and submit it again_**
 
 Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready :pray: 
 
 Cheers and Happy coding! :clap: 
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me @ichala in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**